angular1/README.md

```diff
@@ -0,0 +1,57 @@
+# Angular Input Mask
+
+## Getting started
+
+First, install it.
+
+```bash
+npm i angular1-text-mask --save
+```
+
+Then, use it as follows:
+
+```html
+<script
+  type="text/javascript"
+  src="./node_modules/angular1-text-mask/dist/angular1TextMask.js"></script>
```

Comment:
We could write a test that check if the module was loaded. But usually you do not check if the module is loaded...